scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ClientHandshaker.java

```diff
@@ -471,6 +471,7 @@ private void receivedServerHelloDone(ServerHelloDone message) throws HandshakeEx
 			generateKeys(premasterSecret);
 			break;
 		case PSK:
+			recordLayer.pauseRetransmission();
```

Comment:
For the client side, if we reached this method, I think just `stop `the "own retransmission" would be enought. And we may move this more to the upper, say also for "expensive asymetric" cipher suites. Though very short later, the new flight will cancel the retransmission, I don't think it pays off to "resume/restart" the old retransmission.